src/test/java/com/mael/payroll/transactions/AddTimeCardTest.java

```diff
@@ -0,0 +1,46 @@
+package com.mael.payroll.transactions;
+
+import com.mael.payroll.Employee;
+import com.mael.payroll.PayrollDBFacade;
+import com.mael.payroll.cards.TimeCard;
+import com.mael.payroll.paymentTypes.HourlyPayment;
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class AddTimeCardTest {
+
+    private Employee employee;
+    private PayrollDBFacade payrollDB;
+    private AddTimeCard timeCardTransaction;
+
+    @Before
+    public void setUp() {
+        payrollDB = new PayrollDBFacade();
+    }
+
+    @Test
+    public void addsTimeCardToHourlyEmployee() {
+        AddEmployee addHourlyEmployee = new AddHourlyEmployee(payrollDB, 1, "Squiddo", "FishBowl", 1000.0);
+        addHourlyEmployee.execute();
+        employee = payrollDB.getEmployee(1);
+
+        timeCardTransaction = new AddTimeCard(payrollDB, 1, 10001010, 8.0);
```

Comment:
Woah, you're right... I didn't think too deeply about it because I knew I would be using a real `Date()` object in the future. Anyway, I'm doing exactly that right now (implementing dates) :+1: